CSS/Styles.css

```diff
@@ -380,9 +476,186 @@ textarea {
 .socialinks li {
   display: inline-block;
   padding: 0;
+  width: fit-content;
+  height: auto;
 }
 
 .socialinks img {
   height: 50%;
   width: auto;
 }
+
+@media screen and (min-width: 768px) {
+  .nav {
+    box-shadow: 0 0.5px 0 #dddfe6;
+    background-image: none;
+    justify-content: center;
+    align-items: center;
+  }
+
+  .menu {
+    display: none;
+  }
+
+  .navlinks {
+    display: flex;
+  }
+
+  .homesection {
+    margin-bottom: 60px;
+    flex-wrap: nowrap;
+    height: 100vh;
+  }
+
+  .homewrp,
+  .aboutwrpr {
+    width: 50%;
+    padding-left: 70px;
+    justify-content: flex-start;
+    align-items: flex-start;
+  }
+
+  .maintitle {
+    width: 100%;
+  }
+
+  .mtitle {
+    font-size: 90px;
+    line-height: 90px;
+    padding: 0;
+  }
+
+  .maintxt {
+    width: 90%;
+  }
+
+  .mtext {
+    margin: 0;
+  }
+
+  .decoimg {
+    display: none;
+  }
+
+  .decoimgdsktp {
+    display: block;
+  }
+
+  .works {
+    padding-top: 120px;
+  }
+
+  .project {
+    width: auto;
+    height: 20%;
+    flex: 1 1 25%;
+  }
+
+  .projtitle {
+    background-image: url(../Images/image_dots.svg);
+    background-repeat: no-repeat;
+    background-position: right top;
+    justify-content: center;
+    align-items: flex-start;
+    flex: 1 0 40%;
+  }
+
+  .projtitle h1 {
+    position: relative;
+    transform: rotate(-90deg);
+    width: 100%;
+    text-align: left;
+  }
+
+  .projectwrp {
+    display: none;
+  }
+
+  .mproj {
+    width: auto;
+    flex: 1 1 50%;
+  }
+
+  .mproj .projectwrp {
+    display: block;
+  }
+
+  .shape1 {
+    display: block;
+  }
+
+  .shape2 {
+    display: block;
+  }
+
+  .about {
+    width: 100%;
+    justify-content: center;
+    align-items: center;
+    flex-wrap: nowrap;
+    flex-direction: row-reverse;
+  }
+
+  .aboutwrpr {
+    padding: 0;
+    margin: 0 0 0 60px;
+    width: 50%;
+  }
+
+  .contain {
+    width: 40%;
+  }
+
+  .card {
+    padding: 0;
+    margin: 0;
+    height: 200px;
+    width: 200px;
+    flex: 1 1 40%;
+  }
+
+  .sectitle {
+    font-size: 13px;
+  }
+
+  .litxt ul {
+    margin-top: 0;
+    margin-bottom: 15px;
+    font-size: 11px;
+    font-weight: 200;
+    line-height: 14px;
+  }
+
+  .dots {
+    background-image: url(../Images/image_dots.svg);
+    background-repeat: no-repeat;
+    background-size: 90%;
+    background-position: right bottom;
+    width: auto;
+    height: auto;
+    flex: 1 0 25%;
+  }
+
+  .contact {
+    flex-wrap: nowrap;
+  }
```

Comment:
Your `Contact` section doesn't quite look like the one from the template design.

Here's your design:

<img width="1440" alt="Screen Shot 2022-06-03 at 13 17 53" src="https://user-images.githubusercontent.com/82426682/171923023-af5f73be-d520-4d50-a7f9-1807d44361b9.png">

Here's the template design:

<img width="539" alt="Screen Shot 2022-06-03 at 13 18 23" src="https://user-images.githubusercontent.com/82426682/171923100-f96b0a71-ae9d-4d82-8289-c077ac8f1a92.png">

You should center the section's header and paragraph as well as make the button a little narrower. Kindly fix this.

You need to fix the margin with the section above as well.